test/e2e_test_cluster_image_policy.sh

```diff
@@ -544,3 +544,87 @@ echo '::group::' Cleanup
 kubectl delete cip --all
 kubectl delete ns demo-key-sha512
 echo '::endgroup::'
+
+# Publish the first test image
+echo '::group:: publish test image demoEphemeralImage'
+pushd $(mktemp -d)
+go mod init example.com/demo
+cat <<EOF > main.go
+package main
+import (
+  "fmt"
+  "time"
+)
+func main() {
+  // Calling Sleep method
+  time.Sleep(8 * time.Minute)
```

Comment:
No action necessary, just jotting this down :)

Was worried that this pod would hang around for 8 minutes, but looking at runtimes for the tests that's not the case  😰 🤣

Reply:
The intention is to have a running pod, so we can create an ephemeral one.

Reply:
I could reduce the sleep time to 3min or so, i didn't want to be pick with the time